python/cudf/cudf/tests/test_avro_reader_fastavro_integration.py

```diff
@@ -212,7 +212,22 @@ def test_can_parse_no_schema():
 
 @pytest.mark.parametrize("rows", [0, 1, 10, 1000])
 @pytest.mark.parametrize("codec", ["null", "deflate", "snappy"])
-def test_avro_compression(rows, codec):
+@pytest.mark.parametrize(
+    "env_vars",
+    [
+        {
+            "LIBCUDF_HOST_DECOMPRESSION": "OFF",
+            "LIBCUDF_NVCOMP_POLICY": "ALWAYS",
+        },
+        {"LIBCUDF_HOST_DECOMPRESSION": "OFF", "LIBCUDF_NVCOMP_POLICY": "OFF"},
+        {"LIBCUDF_HOST_DECOMPRESSION": "ON"},
+    ],
+)
```

Comment:
If these are the environment variables you consistently want to test in the avro, orc, and parquet tests, in `conftest.py` you can define.

```python
@pytest.fixture(params=[
    {"LIBCUDF_HOST_DECOMPRESSION": "OFF", "LIBCUDF_NVCOMP_POLICY": "OFF"},
    ...
)
def set_compression_env_vars(monkeypatch, request):
     env_vars = request.param
     with monkeypatch.context() as m:
         for key, value in env_vars.items():
             m.setenv(key, value)
         yield
```

And then in the test function specify

```python
@pytest.mark.parametrize("compression", [...])
def test_orc_decompression(set_compression_env_vars, compression, non_nested_pdf):
    ...
```

Reply:
this is great, thanks for the suggestion!